torch/csrc/lazy/python/init.cpp

```diff
@@ -1,12 +1,86 @@
 #include <torch/csrc/lazy/python/init.h>
 
+#include <c10/core/Device.h>
+#include <torch/csrc/jit/python/pybind.h>
+#include <torch/csrc/lazy/backend/backend_device.h>
 #include <torch/csrc/lazy/core/debug_util.h>
+#include <torch/csrc/lazy/core/lazy_graph_executor.h>
+#include <torch/csrc/lazy/core/metrics.h>
 #include <torch/csrc/lazy/python/python_util.h>
+#include <torch/csrc/lazy/ts_backend/ts_backend_impl.h>
+#include <string>
+#include <vector>
 
 namespace torch {
 namespace lazy {
 
-void initLazyBindings(PyObject* /* module */){
+
+struct NoGilSection {
+  NoGilSection() : state(PyEval_SaveThread()) {}
```

Comment:
can we use `pybind11::gil_scoped_release` ? this is what pytorch seems to recommend